conftest.py

```diff
@@ -76,9 +76,6 @@ def test_bucket():
 
     yield bucket
 
-    # cleanup test bucket
-    bucket.delete_blobs(bucket.list_blobs())
```

Comment:
We don't need to delete all the files here. Instead, tests that access GCS write all their files to `temporary_gcs_folder` which will be removed at the end of the test (see below)